ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -399,8 +399,41 @@ def _router_bgp_vlan_aware_bundles(self) -> list | None:
                     )
 
             # L2 Vlans per Tenant
+            # If multiple L2 Vlans share the same evpn_vlan_bundle name, they will be part of the same vlan-aware-bundle
+            l2vlans = list(tenant["l2vlans"])
```

Comment:
I believe the data has been normalized in `_filtered_tenants` so we don't need to make it a list here.